web/src/copy_and_paste.ts

```diff
@@ -433,6 +433,31 @@ export function paste_handler_converter(paste_html: string): string {
             return "~~" + content + "~~";
         },
     });
+    turndownService.addRule("latexMath", {
+        filter(node: Node) {
+            if (node instanceof Element) {
+                const closestDisplay = node.closest(".katex-display");
+                return closestDisplay !== null;
+            }
+            return false;
+        },
+        replacement(content: string, node: Node) {
+            if (node instanceof Element) {
+                const displayElement = node.closest(".katex-display")! || node;
+                const mathmlElement = displayElement.querySelector(".katex-mathml");
+                if (mathmlElement) {
+                    const annotation = mathmlElement.querySelector(
+                        'annotation[encoding="application/x-tex"]',
+                    );
+                    if (annotation && annotation.textContent) {
+                        const latexContent = annotation.textContent.trim();
+                        return `\`\`\`math\n${latexContent}\n\`\`\`\n`;
+                    }
+                }
+            }
```

Comment:
This seems to only support math blocks, but inline math is also a feature.

Reply:
Ok, Solving this issue in a while.